addons/mail/static/src/new/rtc/call_settings.js

```diff
@@ -0,0 +1,14 @@
+/** @odoo-module */
+
+import { Component } from "@odoo/owl";
+import { useMessaging } from "../messaging_hook";
+
+export class CallSettings extends Component {
+    setup() {
+        this.messaging = useMessaging();
```

Comment:
messaging is not being used inside the component for now

Reply:
it will definitely be used at some point. might as well keep the import, so we don't need to readd it soon